Add World Health Organisation & ZOA to the GB Dashboard - remove AFD

dalepotter opened the pull request:
Adds these two organisations to the GB dashboard.

dalepotter committed:
Add base info data for the World Health Organisation

dalepotter committed:
Add base info data for France AFD

Note the baseline value has initially been set to zero, pending calculation of this publisher's baseline value (as of 1 June 2017).

dalepotter committed:
Add base info data for ZOA

Note the baseline value has initially been set to zero, pending calculation of this publisher's baseline value (as of 1 June 2017).

hayfield commented on data/static/base_info.csv:
```diff
@@ -1,4 +1,5 @@
 name_en,first_published,org_type,registry_id,baseline
+Agence Française de Développement,30-09-2016,Other Public Sector,afd,0
```
The date should be in the format `yyyy-mm-dd`

PetyaKangalova replied:
Thanks! Just to say we are also holding off adding AFD for now.

dalepotter replied:
Date formatting fixed in 9f5a055 (although I have received notification from @PetyaKangalova that AFD will be removed from the Grand Bargain signatories).

hayfield commented on data/static/base_info.csv:
```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 World Vision Netherlands,2016-06-20,National NGO,wvnld,3.25
 World Vision UK,2012-06-13,International NGO,wvuk,22
 World Vision Zimbabwe,2017-12-05,International NGO,wvz-gec,0
+ZOA,10-02-2016,International NGO,zoa,0
```
The date should be in the format `yyyy-mm-dd`

PetyaKangalova replied:
Have added Zoa to title of the pull request to avoid confusion.

dalepotter replied:
Date formatting fixed in 9f5a055.

hayfield reviewed (dismissed):
A couple of mis-formatted dates

dalepotter commented:
This PR was closed automatically as an unintended consequence of the set-up of the `dev` branch without phase 1 changes. It has now been reopened.

dalepotter committed:
Fix date formatting

First publication dates were erronously added in format `dd-mm-yyyy`. This fixes the format to `yyyy-mm-dd` to ensure consistency with other dates in this file.

PetyaKangalova commented on data/static/base_info.csv:
```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 United States,2013-01-02,Government,unitedstates,27.25
 US Agency for International Development (USAID),2017-04-27,Government,usaid,33
 "World Bank, The",2011-03-14,Multilateral,worldbank,51.35
+World Health Organisation,2017-06-08,Multilateral,who,0
```
Waiting for  confirmation on when WHO formally joined the Grand Bargain as a signatory

PetyaKangalova replied:
Confirmation from WHO that they joined  1 Jan 2018 . Please recalculate baseline as of 1 January 2018.

PetyaKangalova commented on data/static/base_info.csv:
```diff
@@ -1,4 +1,5 @@
 name_en,first_published,org_type,registry_id,baseline
+Agence Française de Développement,2016-09-30,Other Public Sector,afd,0
```
France is a signatory but AFD are not publishing humanitarian data so will not be added to the dashboard. Confirmation received from AFD.

PetyaKangalova committed:
All new signatories since June 2017

For Grand Bargain signatories the baseline value needs to be recalculated at the point of the organisations joining the Grand Bargain. Comments with the date of joining are added next to each organisation.

PetyaKangalova commented on data/static/base_info.csv:
```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 World Vision Netherlands,2016-06-20,National NGO,wvnld,3.25
 World Vision UK,2012-06-13,International NGO,wvuk,22
 World Vision Zimbabwe,2017-12-05,International NGO,wvz-gec,0
+ZOA,2016-02-10,International NGO,zoa,0
```
Joined at the end of July 2017. Baseline value should be recalculated for 1 August 2017. It has been agreed that the first of the month will be taken for any new signatories.

PetyaKangalova committed:
All new signatories with baseline values

PetyaKangalova committed:
Update base_info.csv

The baseline values for all new signatories have now been added to this pull request. Please check and push to live.

dalepotter committed:
Merge pull request #82 from devinit/PetyaKangalova-patch-1

All new signatories since June 2017

PetyaKangalova reviewed (approved):
All baseline values are correct as per @wendyrogers 's instructions
